Objective-C/CBLFullTextIndexConfiguration.m

```diff
@@ -37,14 +37,28 @@ - (instancetype) initWithExpression: (NSArray<NSString*>*)expressions
     return self;
 }
 
+- (instancetype) initWithExpression: (NSArray<NSString*>*)expressions
+                      ignoreAccents: (BOOL)ignoreAccents
+                           language: (NSString* __nullable)language
+                              where: (nullable NSString*)where {
+    self = [super initWithIndexType: kC4FullTextIndex
+                        expressions: expressions];
+    if (self) {
+        _where = where;
+        // there is no default 'ignoreAccents', since its NOT an optional argument.
+        _ignoreAccents = ignoreAccents;
+        _language = language;
+    }
+    return self;
+}
```

Comment:
Can you make the initWithExpression:ignoreAccents:language constructor calls this constructor?